examples/docs/astro.config.mjs

```diff
@@ -2,7 +2,7 @@ export default {
   // projectRoot: '.',     // Where to resolve all URLs relative to. Useful if you have a monorepo project.
   // pages: './src/pages', // Path to Astro components, pages, and data
   // dist: './dist',       // When running `astro build`, path to final static output
-  // public: './public',   // A folder of static files Astro will copy to the root. Useful for favicons, images, and other files that don’t need processing.
```

Comment:
I think we can leave the code comments alone as long as they are correct, since they won't get transformed

Reply:
Yeah this one slipped by on accident, I only wanted files in the `docs` folder but this in `examples/docs` which was included cause my search was naïve and also isn't even a markdown file like u said 😄 Definitely a good call

Reply:
Actually just peeking into the file now it does have straight quotes 
`(set to "false" to disable)`
so maybe it should really be a straight quote since I think this is the only non-ascii quote in the file

Reply:
I would actually say keep it, and 😅 rewrite that line as 
```
`false`
```
rather than `"false"` so that it indicates that it is a code value, rather than requiring you to set the value as `"false"`, a string?

Reply:
Though that's more of a cross-example change 😅

Reply:
Fair enough, will undo my change in astro.config.mjs in a bit and not modify it 🙂